pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
```

Comment:
@ofek There's a `gradio/templates` directory that's created when the front-end code is built. We need to include it in the source and wheel distribution. Can you please make the change? Thank you!

Reply:
Done